src/Components/test/E2ETest/Tests/KeyTest.cs

```diff
@@ -33,7 +38,7 @@ protected override void InitializeAsyncCore()
             Navigate(ServerPathBase, noReload: !_serverFixture.UsingAspNetHost);
         }
 
-        [Fact]
+        [Fact(Skip = "https://github.com/dotnet/corefx/issues/37313")]
```

Comment:
This feels like we ought to block the integration on this fix. There are enough tests skipped that I have the heebie-jeebies.